temporian/core/operators/simple_moving_average.py

```diff
@@ -29,11 +30,12 @@ class SimpleMovingAverage(Operator):
     def __init__(
         self,
         event: Event,
-        window_length: str,
+        window_length: Duration,
         sampling: Optional[Event] = None,
     ):
         super().__init__()
 
+        self._window_length = window_length
```

Comment:
How does the `_window_length` attribute integrate with the proto definition? Can attributes be defined outside of the `attributes` attribute? (little bit of a tongue twister, I know 😂 )

Reply:
Any field can be created in the operator.

The current logic is that during serialization, an op object is created with the constructor argument registered with the "add_{attribute,output,inputs}" function. While we don't have such op anymore (placeholder op was one of them), we can have operators where the attributes don't need to be serialized.

Later on, would could imagine improvement to this logic. For example, maybe having all the constructor argument passed to "super().__init__()" and registered there would work.